instance_pool.go

```diff
@@ -0,0 +1,142 @@
+package egoscale
+
+type InstancePoolState string
+
+const (
+	InstancePoolCreating    InstancePoolState = "creating"
+	InstancePoolRunning     InstancePoolState = "running"
+	InstancePoolDestroying  InstancePoolState = "destroying"
+	InstancePoolScalingUP   InstancePoolState = "scaling-up"
```

Comment:
```suggestion
	InstancePoolScalingUp   InstancePoolState = "scaling-up"
```

Reply:
When it's last two letter It's recommended to put it in uppercase

Reply:
For abbreviations such as "ID" yes, not for common words such as "up".

Reply:
https://github.com/golang/go/wiki/CodeReviewComments#initialisms

Yes thanks agree so !

Reply:
Do we agree that "up" is not an initialism or acronym, and thus it should be written `Up` and not `UP`? 😉

Reply:
Yes we are agree :smile: